src/components/CheckoutForm/Confirmation.js

```diff
@@ -170,15 +173,10 @@ const Confirmation = ({
           className={styles.btn}
           fluid
         >
-          Go {name.toUpperCase()} now
+          {isTrialEnd
+            ? `Upgrade to ${name.toUpperCase()}`
+            : 'Start 14-day free trial'}
```

Comment:
I'm not in context, but here looks like we suggest new trial if current trial not ended yet